Snowflake.Data/Core/SFBlockingChunkDownloaderV3.cs

```diff
@@ -136,7 +136,7 @@ private async Task<IResultChunk> DownloadChunkAsync(DownloadContextV3 downloadCo
                 qrmk = downloadContext.qrmk,
                 // s3 download request timeout to one hour
                 RestTimeout = TimeSpan.FromHours(1),
-                HttpTimeout = TimeSpan.FromSeconds(16),
+                HttpTimeout = Timeout.InfiniteTimeSpan, // Disable timeout for each request

```

Comment:
I guess this is the fix for Appveyor?

Reply:
@ChTimTsubasa I am not really sure if this change alone will help. I think both changes might be necessary to get the full effect.